components/local-app/main.go

```diff
@@ -101,7 +110,12 @@ func main() {
 					if c.Bool("mock-keyring") {
 						keyring.MockInit()
 					}
-					return run(c.String("gitpod-host"), c.String("ssh_config"), c.Int("api-port"), c.Bool("allow-cors-from-port"), c.Bool("auto-tunnel"), c.String("auth-redirect-url"))
+					logLevel, err := logrus.ParseLevel(c.String("log-level"))
+					if err != nil {
+						logLevel = logrus.InfoLevel
+						logrus.WithError(err).WithField("logLevel", c.String("log-level")).Error("failed to parse log-level")
+					}
+					return run(c.String("gitpod-host"), c.String("ssh_config"), c.Int("api-port"), c.Bool("allow-cors-from-port"), c.Bool("auto-tunnel"), c.String("auth-redirect-url"), logLevel)
```

Comment:
nit: In most of our CLI's we introduced a `--verbose` flag which sets the log level to `debug`. What's the use-case to set the log level to something other than `debug` or `info`?

Reply:
refactored to verbose